OMCompiler/Compiler/Template/CodegenC.tpl

```diff
@@ -1769,23 +1770,24 @@ template symJacDefinition(list<JacobianMatrix> JacobianMatrices, String modelNam
 ::=
   let symbolicJacsDefine = (JacobianMatrices |> jac as JAC_MATRIX(columns=jacColumn, seedVars=seedVars, matrixName=name, jacobianIndex=indexJacobian)  =>
     <<
-    #if defined(__cplusplus)
-    extern "C" {
-    #endif
-      #define <%symbolName(modelNamePrefix,"INDEX_JAC_")%><%name%> <%indexJacobian%>
-      int <%symbolName(modelNamePrefix,"functionJac")%><%name%>_column(DATA* data, threadData_t *threadData, ANALYTIC_JACOBIAN *thisJacobian, ANALYTIC_JACOBIAN *parentJacobian);
-      int <%symbolName(modelNamePrefix,"initialAnalyticJacobian")%><%name%>(DATA* data, threadData_t *threadData, ANALYTIC_JACOBIAN *jacobian);
-      <%genericCallHeaders(jac.generic_loop_calls, createJacContext(jac.crefsHT))%>
-    #if defined(__cplusplus)
-    }
-    #endif
+    #define <%symbolName(modelNamePrefix,"INDEX_JAC_")%><%name%> <%indexJacobian%> /* PH */
```

Comment:
@phannebohm I think you left your debug comment. I had one at the same location 😆

Reply:
Yes I did. I opened #10378 but it failed for some reason :see_no_evil: